src/main/java/propertyFiles/citation.properties

```diff
@@ -106,7 +106,7 @@ datasetfieldtype.topicClassification.description=The classification field indica
 datasetfieldtype.topicClassValue.description=Topic or Subject term that is relevant to this Dataset.
 datasetfieldtype.topicClassVocab.description=Provided for specification of the controlled vocabulary in use, e.g., LCSH, MeSH, etc.
 datasetfieldtype.topicClassVocabURI.description=Specifies the URL location for the full controlled vocabulary.
-datasetfieldtype.publication.description=Publications that use the data from this Dataset.
+datasetfieldtype.publication.description=Publications that use the data from this Dataset. The full list of Related Publications will be displayed on the metadata tab.
```

Comment:
@JayanthyChengan this looks fine but do you have any insight into the fact that the longer version appears above  while the shorter version appears below? Is this by design? Here's a screenshot:

<img width="988" alt="Screen Shot 2020-05-12 at 4 46 54 PM" src="https://user-images.githubusercontent.com/21006/81743797-4306f980-9470-11ea-9c69-c74ef5b46525.png">

Reply:
@pdurbin :  corrected both issues and pushed the code.  Please verify

Reply:
@JayanthyChengan this looks good but there should be a release note. Please take a look at https://github.com/scholarsportal/dataverse/pull/75 and merge it if you think it looks ok.

Reply:
@pdurbin - I merged the release note.

Reply:
@JayanthyChengan thanks! From your experience, do you think it's necessary to maintain "description" in both the TSV files and the properties files? I hope not because this seems like a maintenance burden. If so I think we should try to move completely to the properties files.

Reply:
@pdurbin - 
Yes, I agree with you.  Along with "description",  I also see "watermark" which can be moved to property files and stays in one place to maintain.  

We have to review the code to verify it's not affecting any other features.  Since I am engaged with other projects, I won't be able to take this for now.  Sorry.

Reply:
@JayanthyChengan no problem. Thanks for your response and this pull request! Have a great weekend!